pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[build-system]
+requires = ["setuptools", "lsst-versions >= 1.3.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lsst-multiprofit"
+description = "Astronomical image and source model fitting code."
+license = {file = "LICENSE"}
+readme = "README.rst"
+authors = [
+    {name="Rubin Observatory Data Management", email="dm-admin@lists.lsst.org"},
+]
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+keywords = ["astronomy", "astrophysics", "lsst"]
+requires-python = ">=3.10.0"
+dependencies = [
+    "astropy",
+    "gauss2d",
```

Comment:
Is this on pypi?

Reply:
No. I suppose it could be as it's a single repo and meson is pip-installable, although I'm not sure if the build configuration meets PyPI's requirements. For now, I configured the [non-eups build](https://github.com/lsst-dm/gauss2d/blob/tickets/DM-41209/python/meson.build#L43) to pip-install the bindings, so that they show up in `pip list` and are recognized as dependencies.